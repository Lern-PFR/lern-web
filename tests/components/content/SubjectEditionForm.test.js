describe('Subject edition form', () => {
	// @TODO: properly test the subject edition form component.
	it('should pass', () => {
		expect(true).toEqual(true);
	});
});
