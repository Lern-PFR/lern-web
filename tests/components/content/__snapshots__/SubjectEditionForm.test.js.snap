// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<Router
  history={
    Object {
      "action": "POP",
      "block": [Function],
      "createHref": [Function],
      "go": [Function],
      "goBack": [Function],
      "goForward": [Function],
      "listen": [Function],
      "location": Object {
        "hash": "",
        "pathname": "/",
        "search": "",
        "state": undefined,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
  staticContext={Object {}}
>
  <SubjectEditionForm
    onSubmit={[MockFunction]}
    subject={
      Object {
        "description": "dummy_subject_description",
        "id": "dummy_subject_id",
        "title": "dummy_subject_title",
      }
    }
  />
</Router>
`;
